testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -177,6 +178,42 @@ def _initSpinnakerLocationParameters(cls, parser, defaults):
              ' is GCE.')
 
   @classmethod
+  def _initAzureOperationConfigurationParameters(cls, parser, defaults):
+    """Initialize arguments for configuring operations for Azure.
+
+      parser: [argparse.ArgumentParser]
+      defaults: [dict] Default binding value overrides.
+         This is used to initialize the default commandline parameters.
+    """
+    parser.add_argument(
+        '--azure_subscription_name', dest='spinnaker_azure_account',
+        help='The name of your subscriptoin.')
+    parser.add_argument(
+        '--azure_subscription_id', dest='spinnaker_azure_subscription_id',
+        help='The subscription id of your subscriptoin.')
+    parser.add_argument(
+        '--azure_tenant_id', dest='spinnaker_azure_tenant_id',
+        help='The AAD tenant id.')
+    parser.add_argument(
+        '--azure_client_id', dest='spinnaker_azure_client_id',
+        help='The service principal id.')
+    parser.add_argument(
+        '--azure_app_key', dest='spinnaker_azure_app_key',
+        help='The service principal secret.')
+    parser.add_argument(
+        '--azure_location', dest='spinnaker_azure_location',
```

Comment:
It looks like you use this for regions. Is it intended to be more than a region? I'm not familiar with Azure but didnt notice "location" as part of the terminology. I think this is going to lead to confusion down the road. I'd suggest s/_location/_region/ or clarify what a location is.